cogs/mmr.py

```diff
@@ -4,33 +4,28 @@
 from discord.ext import commands
 
 from db import database
+from static import EmbedTemplate
```

Comment:
from static.embed_template import EmbedTemplate

Reply:
From what I found after some searching, I believe that should still work without the ```.embed_template```.

Reply:
Yeah it would work, I’m just trying to keep it consistent with the other code I’ve done so far

Reply:
Sure, will do

Reply:
Thanks! Sorry to be picky. I’ve just spent hours and hours thinking about the best way to do things for this bot and have come a little attached to it so want it to be perfect as it can be haha 😅

Reply:
No problem, perfectly understandable 👍